### ISSUE TITLE
Swap keybinding J/K for next/previous tab

### ISSUE THREAD

**Issue by vlcinsky** [ueokande/vim-vixen#667]
### I'm opening this issue because:

- [x] I'll report a bug
- [ ] I'll propose a new feature

#### Steps to Reproduce
- Install Vim Vixen and keep default keybinding untouched.
- Open couple of tabs.
- Try to use J and K to move between tabs.

Problem: by default, J jumps to the right tab, K to the left. This is counter-intuitive as J is on the left and K on the right.

Fix is easy, remapping the keys in the addon, but I wonder, why goes this way by default.

#### System configuration

- Operating system: Debian Buster
- Firefox version: 68.2.0esr (64 bits)
- Vim-Vixen version: 0.25

**Comment by edrex** [comment]
Agreed, vim bindings are all about spatial mapping. J should go left, K right. All other vim browser bindings I've used are mapped this way.

**Comment by ueokande** [comment]
<kbd>J</kbd> (down) and <kbd>K</kbd> (up) mean next and previous (not left and right) for me.

It is worth reconsidering default mapping as you said, because [vimium](https://github.com/philc/vimium#keyboard-bindings) and [VimFx](https://github.com/akhodakivskiy/VimFx/blob/master/documentation/commands.md#j-k) bind are opposite keys.

**Comment by bergmul** [comment]
I also think the default behavior should be switched. It's easy enough to fix though.

**Comment by LeonT-A** [comment]
As a pretty new vim user that is focusing on the vanilla keystrokes, I can understand that `j` would go DOWN a tab while `k` would go UP a tab. Vim ships with vanilla keystrokes but then users go in and modify them (which is just as easy to do here).

It took a minute to get used to it, but after that no problem. As long as I remember that tabs are opened top to bottom I'm good.

**Comment by EtiamNullam** [comment]
While swapping these bindings might make sense for people that don't use vim, it would be confusing for a vim user, just as @LeonT-A explained, so I hope it will never happen. I expect `J` to go forward in the list of tabs and `K` to go backward.

Most importantly it won't match flow of tabs in extensions that add vertical tab lists. This is natural:

![vertical tab list in tree style tab](https://user-images.githubusercontent.com/10875340/172684481-acbf3811-d27a-43d2-8e1d-5409eda5643a.png)
